sc/social/like/browser/templates/metadata.pt

```diff
@@ -1,5 +1,8 @@
-<tal:metas i18n:domain="sc.social.like" tal:condition="view/enabled">
-  <tal:plugin tal:repeat="plugin view/plugins">
-    <tal:replace replace="structure plugin/html" />
-  </tal:plugin>
-</tal:metas>
+<meta name="twitter:card" content="summary_large_image" />
+<meta name="twitter:site" tal:attributes="content string:@${view/via}" />
+<meta property="og:url" tal:attributes="content view/canonical_url" />
+<meta property="og:type" tal:attributes="content view/type" />
+<meta property="og:locale" tal:attributes="content view/language" />
+<meta property="og:title" tal:attributes="content view/title" />
+<meta property="og:description" tal:attributes="content view/description" />
+<meta property="og:image" tal:attributes="content view/image_url" />
```

Comment:
`og:image` must include metadata about the image itself.

Reply:
Done